cl/recap/models.py

```diff
@@ -212,6 +220,50 @@ def print_file_contents(self):
         print(self.file_contents)
 
 
+class EmailProcessingQueue(AbstractDateTimeModel):
+    uploader = models.ForeignKey(
+        User,
+        help_text="The user that sent in the email for processing.",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+    )
+    court = models.ForeignKey(
+        Court,
+        help_text="The court where the upload was from",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+    )
+    filepath = models.FileField(
+        help_text="The S3 filepath to the email and receipt stored as JSON text.",
+        upload_to=make_recap_email_processing_queue_aws_path,
+        storage=IncrementingAWSMediaStorage(),
+        max_length=300,
+        null=True,
+    )
+    status = models.SmallIntegerField(
+        help_text="The current status of this upload. Possible values "
+        "are: %s"
+        % ", ".join(
+            ["(%s): %s" % (t[0], t[1]) for t in PROCESSING_STATUS.NAMES]
+        ),
+        default=PROCESSING_STATUS.ENQUEUED,
+        choices=PROCESSING_STATUS.NAMES,
+        db_index=True,
+    )
+    status_message = models.TextField(
+        help_text="Any errors that occurred while processing an item",
+        null=True,
```

Comment:
Sorry I missed this before. This should be `blank=True`, not `null=True`, right?

Reply:
Correct. Updating.